scripts/deployment/tvm-deploy/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.6)
+project(gluoncv_lite)
+
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -ldl -pthread")
+#SET(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
+#SET(CMAKE_LIBRARY_OUTPUT_DIRECTORY  ${CMAKE_CURRENT_SOURCE_DIR})
+SET(HOME_TVM ${CMAKE_CURRENT_SOURCE_DIR}/tvm)
+
+INCLUDE_DIRECTORIES(${HOME_TVM}/include)
+INCLUDE_DIRECTORIES(${HOME_TVM}/3rdparty/dmlc-core/include)
+INCLUDE_DIRECTORIES(${HOME_TVM}/3rdparty/dlpack/include)
+
+set(PNG_LIBS libpng.a)
```

Comment:
I can't find libpng, libz, libjpeg anywhere in the PR so this much be something hardcoded.

Reply:
Since we want to build libjpeg, libpng statically and link to them, these libraries should be prepared before building the full project. I didn't provide these files because they depend on specific operating system. I write some comments about the procedure in README and thought developers would build these libraries according to their target platform.  Could you please give me some suggestions on how to make changes? Thanks.

Reply:
The ideal way is to provide the build system for libjpeg and libpng in a cross-platform way(cmake).
We can include the source of libpng/jpeg in this case. Build for static linking, and finally include them in this demo probject CMakeList.

Reply:
Since this is using CMake, Why not use `find_package()` for these?